app/mailers/super_admin_mailer.rb

```diff
@@ -2,10 +2,19 @@ class SuperAdminMailer < ApplicationMailer
   default from: 'remote-flow@mail.com'
 
   def new_organization_email(organization)
-    admins_email = User.super_admins.pluck(:email)
-    return false if admins_email.empty?
-
     @organization = organization
-    mail(to: admins_email, subject: 'New organization created')
+    mail(to: super_admin_emails, subject: 'New organization created') unless super_admin_emails.empty?
```

Comment:
method has small performance issue, multiple querying for super admin emails
prev version was better, except there's no need to explicitly return `false`, `nil` would work better